app/view_objects/assessor_interface/application_forms_show_view_object.rb

```diff
@@ -47,14 +47,18 @@ def assessment_task_path(section, item, index)
         item,
       )
     when :recommendation
-      status = assessment_task_status(section, item, index)
-
-      return nil unless status == :not_started
-
-      url_helpers.edit_assessor_interface_application_form_assessment_path(
-        application_form,
-        assessment,
-      )
+      return nil unless assessment.sections_finished?
+
+      if assessment.unknown? ||
```

Comment:
Could we extract this into a method? 😵‍💫

Reply:
Good point, I've amended the commit to put this in a method.